EmotiBitOscilloscope/src/ofApp.cpp

```diff
@@ -11,6 +11,8 @@ void ofApp::setup() {
 	ofSetLogLevel(OF_LOG_NOTICE);
 	writeOfxEmotiBitVersionFile();
 	setTypeTagPlotAttributes();
+	//saveEmotiBitCommSettings();
```

Comment:
Should we remove this un-used comment? or perhaps add another comment explaining its use?